great-idea/css/index.css

```diff
@@ -115,21 +122,23 @@ h1 {
 .main-header{
     padding: 30px 0;
 }
-
+.section{
+    display: flex;
+    justify-content:space-between;
+}
+.sectiion2{
+    display: flex;
+}
 .product {
-    display: inline-block;
-    width: 270px;
-    margin: 20px 20px 20px 20px;
+
+
 }
 .services {
-    display: inline-block;
-    width: 270px;
-    margin:20px 20px 20px 0px;
```

Comment:
When you refactor items to flex, the individual selectors no longer need `display: inline-block;` since the parent container will be taking care of positioning. However, flex doesn't really do anything to the box model, so you still need these width and margin properties.